spec/controllers/v1/analyses_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe V1::AnalysesController do
+  render_views
+
+  let(:inventory) { FactoryGirl.create(:inventory) }
+
+  before :each do
+    request.env['HTTP_ACCEPT'] = 'application/json'
```

Comment:
Can this be done by using `format: :json` in `post :create, inventory_id: inventory.id, format: :json`?